stdlib/Random/test/runtests.jl

```diff
@@ -686,3 +686,11 @@ end
     @test Random.gentype(Random.UInt52(UInt128)) == UInt128
     @test Random.gentype(Random.UInt104()) == UInt128
 end
+
+@testset "shuffle[!]" begin
+    a = []
+    @test shuffle(a) == a # issue #28727
```

Comment:
For the future, putting the issue names in the testset name might be a good idea because you can then see from the test failure the relevant issue.